client/starwhale/utils/config.py

```diff
@@ -108,10 +108,26 @@ def __init__(self, swcli_config: t.Optional[t.Dict[str, t.Any]] = None) -> None:
     def rootdir(self) -> Path:
         return Path(self._config["storage"]["root"])
 
+    @property
+    def workdir(self) -> Path:
+        return self.rootdir / "workdir"
+
+    @property
+    def pkgdir(self) -> Path:
+        return self.rootdir / "pkg"
+
+    @property
+    def dataset_dir(self) -> Path:
+        return self.rootdir / "dataset"
+
     @property
     def datastore_dir(self) -> Path:
         return self.rootdir / ".datastore"
 
+    @property
+    def eval_run_dir(self) -> Path:
```

Comment:
the `eval_run_dir` is wrong location. Is it in the project dir?

Reply:
removed